src/CoreBundle/Session/ChameleonSessionManager.php

```diff
@@ -175,4 +182,21 @@ public function isSessionStarting()
     {
         return $this->isSessionStarting;
     }
+
+    /**
+     * @return string
+     */
+    private function getCookieNameSuffix()
+    {
+        if (!CHAMELEON_EXTRANET_USER_IS_PORTAL_DEPENDANT) {
```

Comment:
Can we assume that the constant is always defined or would adding a `defined(...)` check make sense?

Reply:
Yes, we've got a default in every project in `/src/CoreBundle/Resources/config/defaults.inc.php:651`